DataFormats/TrackerCommon/interface/LegacyTrackerTopology.h

```diff
@@ -0,0 +1,98 @@
+#ifndef TRACKER_TOPOLOGY_LEGACY_H
+#define TRACKER_TOPOLOGY_LEGACY_H
+
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "DataFormats/DetId/interface/DetId.h"
+
+#include <memory>
+
+// WARNING: this header has been introduced to call a TrackerTopology
+// object whenever it is not possible to access it through an EventSetup.
```

Comment:
Hi @slava77, a couple of use cases are:
   * from within a ROOT macro (as in  `CalibTracker/SiStripChannelGain/test/UsefullCode/TimeHistory/PlotMacro.C`)
   * from a payload inspector application (see discussion at: https://github.com/cms-sw/cmssw/pull/19193#issuecomment-310304097 and following) 

The proposed implementation is actually inspired by David's https://github.com/cms-sw/cmssw/pull/19393#issuecomment-310397165, though I am not sure this is exactly what he had in mind at the time.